contrib/systemd/usbsdmux.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=USB-SD-Mux Proxy-Service
+After=network.target usbsdmux.socket
+Requires=usbsdmux.socket
+
+[Service]
+Type=simple
+ExecStart=/path/to/usb-sd-mux/.venv/bin/usbsdmux-service
+TimeoutStopSec=5
```

Comment:
Why do we need to reduce TimeoutStopSec?

Reply:
You are right. We can use the system default for TimeoutStopSec here.
I'll remove the TimeoutStopSec parameter.